diff --git a/package/lean/autocore/files/x86 b/package/lean/autocore/files/x86
index dcaaa55..34ae9e9 100644
--- a/package/lean/autocore/files/x86
+++ b/package/lean/autocore/files/x86
@@ -60,7 +60,7 @@
 			ethinfo    = eth_info,
 			userinfo    = user_info,
 			uptime     = sysinfo.uptime or 0,
-			localtime  = os.date(),
+			localtime  = os.date("%Y-%m-%d") .. " " .. translate(os.date("%A")) .. " " .. os.date("%X"),
 			loadavg    = sysinfo.load or { 0, 0, 0 },
 			memory     = meminfo,
 			memcached  = mem_cached,
@@ -188,20 +188,6 @@
 		});
 	}
 
-	var compare = function (prop) {
-		return function (obj1, obj2) {
-			var val1 = Number(obj1[prop].replaceAll(".",""));
-			var val2 = Number(obj2[prop].replaceAll(".",""));
-			if (val1 < val2) {
-				return -1;
-			} else if (val1 > val2) {
-				return 1;
-			} else {
-				return 0;
-			}
-		}
-	};
-
 	XHR.poll(1, '<%=REQUEST_URI%>', { status: 1 },
 		function(x, info)
 		{
@@ -395,7 +381,6 @@
 			var ls = document.getElementById('lease_status_table');
 			if (ls)
 			{
-				info.leases = info.leases.sort(compare("ipaddr"));
 				/* clear all rows */
 				while( ls.rows.length > 1 )
 					ls.rows[0].parentNode.deleteRow(1);
@@ -740,17 +725,24 @@
 		<tr><td width="33%"><%:Model%></td><td><%=pcdata(boardinfo.model or "?")%> <%=luci.sys.exec("cat /etc/bench.log") or " "%></td></tr>
 		<tr><td width="33%"><%:CPU Info%></td><td id="cpuinfo">-</td></tr>
 		<tr><td width="33%"><%:Firmware Version%></td><td>
-			<%=pcdata(ver.distname)%> <%=pcdata(ver.distversion)%> /
-			<%=pcdata(ver.luciname)%> (<%=pcdata(ver.luciversion)%>)
+			<%=pcdata(ver.distname)%> <%=pcdata(ver.distversion)%><!-- /
+			<%=pcdata(ver.luciname)%> (<%=pcdata(ver.luciversion)%>)-->
 		</td></tr>
 		<tr><td width="33%"><%:Kernel Version%></td><td><%=unameinfo.release or "?"%></td></tr>
 		<tr><td width="33%"><%:Local Time%></td><td id="localtime">-</td></tr>
 		<tr><td width="33%"><%:Uptime%></td><td id="uptime">-</td></tr>
 		<tr><td width="33%"><%:Load Average%></td><td id="loadavg">-</td></tr>
 		<tr><td width="33%"><%:CPU usage (%)%></td><td id="cpuusage">-</td></tr>
+</table>
+</fieldset>
+<fieldset class="cbi-section">
+                                <td id="cpr">
+<input type="button" class="cbi-button cbi-button-apply" value="<%:更新教程%> " onclick="window.open('https://github.com/shidahuilang/openwrt')" />
+<input type="button" class="cbi-button cbi-button-apply" value="<%:固件下载%> " onclick="window.open('https://github.com/shidahuilang/openwrt/releases')" />
 	</table>
 </fieldset>
 
+
 <fieldset class="cbi-section">
 	<legend><%:Memory%></legend>
 
@@ -772,6 +764,8 @@
 <% end %>
 
 <fieldset class="cbi-section">
+<iframe allowtransparency="true" frameborder="0" width="1100" height="86" scrolling="no" src="//tianqi.2345.com/plugin/widget/index.htm?s=2&z=1&t=0&v=0&d=5&bd=0&k=000000&f=&ltf=009944&htf=cc0000&q=1&e=1&a=1&c=57083&w=575&h=96&align=right"></iframe>
+
 	<legend><%:Interfaces%></legend>
 
 	<table width="100%" cellspacing="10" id="ethinfo">
